### ISSUE TITLE
cannot import bookmarks file, maybe timerangeerror?

### ISSUE THREAD

**Issue by Neurofibromin** [linkwarden/linkwarden#889]
**Describe the bug**
When importing from a bookmarks.html file I get the error message: An error occured while importing bookmarks. Please check the logs for more info.

**To Reproduce**
I have freshly exported my bookmarks from google chrome to html, when importing in the settings the above message is shown. Nothing is imported. 

**Expected behavior**
Import to work. It does so with a much smaller file from Firefox.

**Desktop (please complete the following information):**
 - OS: Fedora Linux
 - Browser chrome & Firefox
 - Version Latest stable everything

**Additional context**
Self hosting with docker, using the default settings in the docker compose file. Firefox import works fine for the smaller file.

Docker Logs:

[logs.txt](https://github.com/user-attachments/files/17893717/logs.txt)

I also use many utf-8 characters in folder and bookmark names, as well as empty folder names but I found GH issues saying those are supported.

Thanks for any help!

**Comment by daniel31x13** [comment]
Yes the date in the bookmark file is most likely improperly formatted. Convert the bookmark dates to [Unix timestamp](https://www.unixtimestamp.com) (in seconds) and it'll be fixed.

**Comment by Neurofibromin** [comment]
Actually it was not a timestamp issue, no idea why that was in the logs. Setting the import limit to higher solved the issue for me.
Import limit:
https://github.com/linkwarden/linkwarden/issues/99
